test/wilson/dom_test.cljs

```diff
@@ -114,12 +141,12 @@
                                    "j" {:class "warning"}
                                    "r" {:class "success"}
                                    {:class "info"}))})
-           [:table {:class "table table-hover"}
+           [:table {:class "table"}
```

Comment:
That class isn't currently configurable; but yeah, with-class works. Thanks for updating shrieker :)